src/config.rs

```diff
@@ -24,3 +25,15 @@ impl Config {
         cfg.try_into()
     }
 }
+
+pub const ARGON2_CONFIG: argon2::Config<'_> = Config {
```

Comment:
Бесполезное использование префикса пути
```suggestion
pub const ARGON2_CONFIG: Config<'_> = Config {
```